packages/actor-abstract-bindings-hash/test/AbstractBindingsHash-test.ts

```diff
@@ -83,10 +83,10 @@ describe('AbstractBindingsHash', () => {
     it('should run', () => {
       const op = { operation: { type: 'distinct' } };
       return m.run(op).then(async (output: IActorQueryOperationOutputBindings) => {
-        expect(await output.metadata()).toEqual({ totalItems: 5 });
-        expect(output.variables).toEqual([ 'a' ]);
-        expect(output.type).toEqual('bindings');
-        expect(await arrayifyStream(output.bindingsStream)).toEqual([
+        void expect(await output.metadata()).toEqual({ totalItems: 5 });
+        void expect(output.variables).toEqual([ 'a' ]);
+        void expect(output.type).toEqual('bindings');
+        return expect(await arrayifyStream(output.bindingsStream)).toEqual([
```

Comment:
(very minor) I would `void` them all for consistency; the return value of `expect` is never used.